statsmodels/tests/test_granger_error_string.py

```diff
@@ -0,0 +1,11 @@
+# this will trigger an InfeasibleTestError exception in 9 lags
+from statsmodels.tsa.stattools import grangercausalitytests
+import numpy as np
+
+X = [1,2,3,4,5,6,7,8,9,10] * 100
+X = [float(x) for x in X]
+Y = [x+1 for x in X]
+
+both = [(a,b) for a,b in zip(X,Y)]
+
+grangercausalitytests(both, maxlag=9)
```

Comment:
You need to actually trigger the error, and then use `pytest.raises` to catch it and finally the `match` argument to match the string.  Or you can just delete this file as doc fixes dont' need tests.

Reply:
Thank you! I wondered if this qualified as a doc fix since it's in the code. I'll delete the test file.